.circleci/config.yml

```diff
@@ -580,6 +580,29 @@ jobs:
               . ~/emsdk/emsdk_env.sh
               # Generate JS CodeConv
               npm run test_with_coverage
+      - run:
+          name: Upload test coverage
+          command: |
+              export PATH=$HOME/miniconda/bin:/usr/local/cuda/bin:$PATH
+              . activate habitat; cd habitat-sim
+
+              curl -Os https://uploader.codecov.io/latest/linux/codecov
+              chmod +x codecov
+              #Uploading test coverage for Python code
+              ./codecov -f coverage.xml -cF Python
+
+              # Uploading test coverage for JS code
+              ./codecov coverage_js/coverage-final.json -cF JavaScript
+
+              #Uploading test coverage for C++ code
+              lcov --directory . --capture --output-file coverage.info
+              # Replaces -1 linecount with zero to prevent lcov from crashing:
+              # https://github.com/psycofdj/coverxygen/issues/6
+              sed -i -e 's/,-1$/,0/g' coverage.info
+              #lcov --remove coverage.info "*/deps/*" --output-file coverage.info > /dev/null
```

Comment:
Quick question, we hy are these commented out? Don't we still not want dependencies to count towards coverage? At the very lest we should add these globs to .coveragerc then.

Reply:
I just copied what was there from the other part :) I don't want to touch anything here, only want to fix what's broken.

Will use the yaml anchor, thanks.